stdlib/template/configure.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python3.6
+# -*- coding: utf-8 -*-
+"""Provides a small, partial template that wraps the ``configure`` command."""
+
+import os
+import stdlib
+from typing import List
+
+
+def get_dir_flags() -> List[str]:
+    """Build a list of directory flags used by the :py:func:`.configure` template.
```

Comment:
```suggestion
    """Builds a list of directory flags used by the :py:func:`.configure` template.
```

Reply:
The current coding style for NBuild is to use infinitive verb to describe a function